SPECS/bcc/bcc.spec

```diff
@@ -1,32 +1,30 @@
 %{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
```

Comment:
Nit: The `python3_sitelib` and `__python3` macro redefinitions can be removed here- we define these in the `mariner-rpm-macros` package.